Makefile

```diff
@@ -0,0 +1,27 @@
+
+SHELL := /bin/bash
+
+.SILENT: clean
+.IGNORE: clean
+
+build:	clean
+	python -m build
+
+clean:
+	rm -rf build/
+	rm -rf dist/
+	rm -rf kafe2.egg-info/
+	rm -rf venv/
+	rm -rf `find . -type d -name __pycache__`
+	#cd docs && $(MAKE) clean
+
+upload: build
+	twine upload dist/*
+
+venv:	build #docs
+	python -m venv venv/
+	. venv/bin/activate; pip install dist/kafe2*.tar.gz
+	. venv/bin/activate; pip install ptpython
```

Comment:
We could add a development venv like this:
```
python -m venv venv/
. venv/bin/activate; pip install -e .[dev]
```

Reply:
Yes, this is a todo we already thought about.